x-pack/plugins/ml/public/application/explorer/explorer_utils.ts

```diff
@@ -23,14 +23,15 @@ import {
   type MlEntityField,
   type MlInfluencer,
   type MlRecordForInfluencer,
+  ML_JOB_AGGREGATION,
 } from '@kbn/ml-anomaly-utils';
 
+import { InfluencersFilterQuery } from '@kbn/ml-anomaly-utils';
```

Comment:
nig: import type

Reply:
Fixed in https://github.com/elastic/kibana/commit/ac4edf9ce9506fd3dfd1c8e43efb665a589ac0f4.